docs/docs/setting-up-ci-cd.mdx

```diff
@@ -36,112 +37,71 @@ resource-intensive tests only when certain files have been changed or when some
+may need a full NLU test with cross validation to look out for confusion between intents.
+
+After testing and posting your results for your team to see, have a person or team of reviewers
+to check the test results with your developer. An easy process to help this is to have your CI
+pipeline automatically create a pull request with the test results in a comment. That way, the
+reviewer can see the changes along with the improved results.
+
+Once the reviewer merges you can then initiate your CD pipeline to add the new model
+to another environment.
+
+### GitHub Actions CI Pipeline
+
+You can use the [Rasa Train-Test Github Action](https://github.com/RasaHQ/rasa-train-test-gha)
+in your CI pipeline to automatically perform data validation, training, and testing.
+
+An example CI pipeline using the Github Action is shown below:
+
+```yaml
+jobs:
+  training-testing:
+    name: Training and Testing
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Rasa Train and Test GitHub Action
+        uses: RasaHQ/rasa-train-test-gha@main
+        with:
+          requirements_file: requirements.txt
+          test_type: all
+          cross_validation: true
+          publish_summary: true
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Upload model
+        if: github.ref == 'refs/heads/master'
+        uses: actions/upload-artifact@master
+        with:
+          name: model
+          path: models
 ```
 
-Training a model verifies that your NLU pipeline and policy configurations are
-valid and trainable, and provides a model to use for testing your assistant.
-If the model passes the CI tests, then you can also [upload the trained model](./setting-up-ci-cd.mdx#deploying-your-rasa-model)
-to your server as part of the continuous deployment process.
+In this pipeline, the Rasa Train-Test Github Action is performing data validation, model training, and story testing
+in the first step and the model file is uploaded as an artifact in the second step.
 
+The complete list of configurable parameters for the Rasa Train-Test Github Action is available in the repository's
+[README](https://github.com/RasaHQ/rasa-train-test-gha#input-arguments).
 
-### Testing Your Assistant
+When `publish_summary` is set to `true`, this action will automatically publish the model's test results to the associated
+Pull Request as a comment:
 
-Testing your trained model on [test stories](./testing-your-assistant.mdx#writing-test-stories) is the best way to have confidence in how your assistant
-will act in certain situations. Written in a modified story
-format, test stories allow you to provide entire conversations and test that, given certain
-user input, your model will behave in the expected manner. This is especially
-important as you start introducing more complicated stories from user
-conversations.
+<img alt="image" src={useBaseUrl("/img/train-test-github-action.png")} />
```

Comment:
```suggestion
<img alt="image" src={useBaseUrl("/img/train-test-github-action.png")} />

After the pull request is opened, tag a reviewer from your team to see the results. The reviewer can then approve the change if the results from the tests are positive, allowing you to continue to Continuous Deployment.
```

Reply:
@b-quachtran looks good, just added a comment to wrap it up at the end.

Reply:
I would err on the side of being less specific here about tagging and reviewing since it's dependent on their individual review processes. Wdyt about this instead:

```suggestion
<img alt="image" src={useBaseUrl("/img/train-test-github-action.png")} />

The pull request can be approved or denied based on the evaluation results and, in many cases, you will want to automate the model's deployment if all CI checks pass. You can continue to the next section to learn more about Continuous Deployment. 
```

Reply:
looks great!